packages/components/src/components/button/button.tsx

```diff
@@ -1,11 +1,41 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import React from 'react';
-import ButtonLoading from './button_loading.jsx';
+import React, { HTMLProps, ReactNode } from 'react';
```

Comment:
```suggestion
import React from 'react';
```

Reply:
@yauheni-kryzhyk-deriv, I'm using `ReactNode` in so many places. Don't you think it'll be so repetitive to prepend`React.`?

Reply:
Actually, I’m not against using such approach. I just propose to stay with the same style in whole project

Reply:
please see here
https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules